module/addItem.js

```diff
@@ -0,0 +1,40 @@
+const nameInput = document.querySelector('.name');
+const scoreInput = document.querySelector('.score');
+const myGame = 'Zl4d7IVkemOTTVg2fUdz';
+const resieveData = async () => {
+  try {
+    const endpoint = new URL(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:id/scores'
+    );
+    const response = await fetch(endpoint);
+    const data = await response.json();
+    return data;
+  } catch (err) {
+    return false;
+  }
+};
+
+const sendData = async () => {
+  try {
+    const sentData = await fetch(
+      `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${myGame}/scores`,
+      {
+        method: 'POST',
+        body: JSON.stringify({
+          user: nameInput.value,
+          score: scoreInput.value,
+        }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      }
+    );
+    nameInput.value = '';
+    scoreInput.value = '';
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job using the leaderboard API! 💯 However, I think these lines that serve as "resetting" the form can be better moved to the submit event listener, this way, your `sendData` function only has the responsibility to send the data to the API, and not also clear the input fields of the form! 🤓 🔧